Brandon's PCB Dog Tag

bseo2002 opened the pull request:
thank you for all the help throughout my struggles!

bseo2002 committed:
Brandon's PCB Dog Tag

thank you for all the help throughout my struggles!

liambontkes reviewed (changes requested):
Your schematic and pcb docs are associated with the project.

fcrema reviewed (changes requested):
It looks like your PCB still has to be finished!

bseo2002 committed:
new commit, can you see the files?

bseo2002 committed:
Gerber File for Brandon PCB DogTag

Please let me know if there is any issues with this. Sorry for the delay

bseo2002 committed:
Upon creating my gerber file, my pcb file said it was locally modified as well, so I am just recommitting the changes

liambontkes commented:
R1 is missing a net

liambontkes committed:
added jlc fabrication output job

bseo2002 committed:
recommit after changes

R1 was missing a net, i have fixed that issue. Please let me know if there are still issues!

bseo2002 committed:
recommit with gerber files!

please let me know if there are any more issues with this project!

liambontkes committed:
produced fabrication files
